test/CMakeLists.txt

```diff
@@ -40,6 +40,15 @@ macro(onedpl_add_test test_source_file)
         target_compile_options(${_test_name} PRIVATE $<$<CONFIG:Release>:-fno-strict-aliasing>)
     endif()
 
+    # Avoid speculative compilation of kernels which may not be supported by a device
+    # This options must be used with DPCPP backend, which requires -fsycl option
+    if (ONEDPL_BACKEND MATCHES "^(dpcpp|dpcpp_only)$")
```

Comment:
May be required to check ${split_kernels} at first ?

Reply:
It is irrelevant for the latest commit. I have removed `split_kernels` argument to enable `-fsycl-device-code-split` for all the tests.